lib/momento/simple_cache_client.rb

```diff
@@ -175,11 +214,12 @@ def delete_cache(cache_name)
 
     # List a page of your caches.
     #
-    # @note Consider using `caches` instead.
-    #
     # The next_token indicates which page to fetch.
     # If nil or "" it will fetch the first page. Default is to fetch the first page.
     #
+    # @see #caches
+    # @see Momento::ListCachesResponse
+    # @note Consider using `caches` instead.
```

Comment:
I probably missed explanations for this in your previous PRs, but why do we have both `list_caches` and `caches`?

Reply:
`list_caches` is the basic GRPC request. It's complicated to use; the caller has to do their own pagination. It's notable we don't have [any API examples for it](https://docs.momentohq.com/API-reference#list-caches). But we want to make the low-level request available.

I wrote `caches` to make it easy. It handles all the pagination.

Chris and I agreed to avoid adding Ruby-specific features to SimpleCacheClient, but it was simpler to write `caches` than try to document how to use `list_caches` and risk users getting it wrong.

Reply:
I'll make that a little more clear.